index.html

```diff
@@ -3018,17 +3071,20 @@ <h3><code>title(s)</code> and <code>description(s)</code></h3>
 
   <p>
   The <a>vocabulary terms</a> <code>titles</code> and <code>descriptions</code>
```

Comment:
The <a>Vocabulary Terms</a> ...

Reply:
Huh.  Not even sure that was my text, but I will fix it.

Reply:
Upon searching for where you noted this issue I notice a bunch of other places where Vocabulary Terms was not upper-cased and/or anchored.   Since I assume this was supposed to be a global convention for special terminology I updated them all (at least the ones in index.template.html; there may be others pulled in as part of rendering).